utils/mermaid.html

```diff
@@ -0,0 +1,57 @@
+<style>
+#options {
+  border-collapse: collapse;
+  width: 100%;
+}
+#options td, #options th {
+  border: 1px solid #ddd;
```

Comment:
Is it same as #0d0d0d or #000ddd?

Reply:
It means #dddddd.
Please refer to the below link's [3 Digit HEX Value] part.
https://www.w3schools.com/css/css_colors_hex.asp

Reply:
Ok, thanks!